gateways/authorizenet/request_builders.go

```diff
@@ -15,7 +15,8 @@ const (
 
 // Options
 const (
-	customerIPOption = "CustomerIP" // Pass as a string pointer
+	customerIPOption                 = "CustomerIP"                 // Pass as a string pointer
+	shouldVoidUnsettledCaptureOption = "ShouldVoidUnsettledCapture" // Pass as a bool
```

Comment:
I'm a bit worried about sleet clients not understanding that this is potentially a different behavior than the refund they tried to submit if the refund was for less than the total amount of the transaction. At minimum we should add a comment describing how the behaviors different, but I'm still not totally convinced that we should even introduce this into sleet. It seems like processing the response code and determining to submit a void should happen in the client's business logic rather than in sleet just to keep sleet less opinionated.